ReportCardGeneration.c

```diff
@@ -0,0 +1,259 @@
+#include<stdio.h>
+int main()
+{
+int i,a,b,c,d,e,n;
+float f,cgpa;
+char name[20];
+printf("***********************************************************\n\n\n");
+printf("Welcome to Student Report Card System\n");
+ 
+ 
+printf("\n\n\n***********************************************************\n");
+ 
+printf("Enter number of students in class\n");
+scanf("%d",&n);
+printf("Enter the Roll Number of student whose report card is required\n");
+scanf("%d",&i);
+switch (i)
+{
+case 1:
+{
+int a=99,b=98,c=97,d=96,e=96;
+f=(float)(a+b+c+d+e)/5;
+cgpa=(float)f/10;
+printf("Name :Alisha \n");
+printf("The Marks obtained by student are as follows :-\n");
+printf("%d\n%d\n%d\n%d\n%d\n",a,b,c,d,e);
+printf("The C.G.P.A obtained is %f \n",cgpa);
+printf("The grade obtained is \n");
+if(cgpa>9)
+{
+printf("A+ grade\n");
+printf("Excellent performance\nKeep that up!\n");
+}
+else if(cgpa>8)
+{
+printf("A grade\n");
+printf("Very Good\n Keep it up!\n");
+}
+else if(cgpa>7)
+{
+printf("B+ grade\n");
+printf("Good performance\n");
+}
+else if(cgpa > 6)
+{
+printf("B Grade\n");
+printf("Good performance. Can be improved!\n");
+}
+else if(cgpa > 5)
+{
+printf("C Grade\n");
+}
+else if(cgpa > 4)
```

Comment:
```suggestion
else if(cgpa > 4)
printf("D Grade\n");
```
Remove extra brackets